makefiles/kconfig.mk

```diff
@@ -36,6 +36,11 @@ KCONFIG_USER_CONFIG = $(APPDIR)/user.config
 # one that is used to generate the 'riotconf.h' header
 KCONFIG_MERGED_CONFIG = $(GENERATED_DIR)/merged.config
 
+# Include configuration symbols if available, only when not cleaning
```

Comment:
>So far, if users want to use Kconfig they will have to call make clean && make all. Any proposal to solve this is appreciated :-)

let's put the problem description in short here into the comment .. I did not come up with a solution for that particular problem, but since it only affects `kconfig` configurations and not other parts of the build system, we can safely continue.

Reply:
I extended the comment.